From 57cf9fce530d1f1fe077d33721ca17169428f103 Mon Sep 17 00:00:00 2001
From: Dan Smith <dasmith@redhat.com>
Date: Thu, 8 Jan 2015 11:41:07 -0500
Subject: [PATCH] Fix typo in reconnect exception handler

Resolves: rhbz#1175685

Signed-off-by: Lon Hohberger <lhh@redhat.com>
---
 oslo/messaging/_drivers/impl_rabbit.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/oslo/messaging/_drivers/impl_rabbit.py b/oslo/messaging/_drivers/impl_rabbit.py
index bdb6693..12a6adf 100644
--- a/oslo/messaging/_drivers/impl_rabbit.py
+++ b/oslo/messaging/_drivers/impl_rabbit.py
@@ -615,7 +615,7 @@ class Connection(object):
                 # a protocol response.  (See paste link in LP888621)
                 # So, we check all exceptions for 'timeout' in them
                 # and try to reconnect in this case.
-                if 'timeout' not in six.text_type(e):
+                if 'timeout' not in six.text_type(ex):
                     raise
                 e = ex
 
